### ISSUE TITLE
class SelectFilteredResult<T> gives InvalidCastException for serializable class

### ISSUE THREAD

**Issue by 0x932b** [NYU-hololens/Hololens-visual#28]
The serializable class SelectFilteredResult which reads a table from AzureSQL cannot be cast to be serializable under a hololens build.
Need to rewrite or fix the current DB access methods.

## Acceptance Criteria:
```
Given
When
Then 
```

**Comment by 0x932b** [comment]
The current public class SelectFilteredResult<T> results in an
InvalidCastException for AzureServicesForUnity.AppService.SelectFilteredResult<1tableA> to Unity.Iserializable.
This Error occurs both as a sphere action and an UI script.

**Comment by 0x932b** [comment]
Postponed for now, waiting on new db
